plugins/workload-management/src/main/java/org/opensearch/plugin/wlm/rule/sync/package-info.java

```diff
@@ -0,0 +1,11 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+/**
+ * This package contains synchronization related constructs
+ */
+package org.opensearch.plugin.wlm.rule.sync;
```

Comment:
The package description is confusing. One might assume concurrency related synchronization

Reply:
In the end we are synchronising the in-memory view of Rules with on disk one. Does adding little more context help here ?
I could think this is a more meaningful package name given the context for the constructs in this package. Can you also put your suggestions ?